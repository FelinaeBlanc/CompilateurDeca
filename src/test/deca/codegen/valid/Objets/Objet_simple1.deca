class A {
    int x = 0;
    int y = 0;
}

{
    int x = 1;
    int y = 2;
    A a = new A() ;
    a.x = 3;
    a.y = 4;

    println("a.x = ", a.x) ;
    println("a.y = ", a.y) ;
}