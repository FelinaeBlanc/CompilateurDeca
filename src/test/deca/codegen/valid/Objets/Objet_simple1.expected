a.x = 3
a.y = 4
